### PR TITLE
Add `body` to the API response

### PR DESCRIPTION
**Pull request by ericholscher**
This is a text-only version of the ad for now.

### CODE REVIEW

File: adserver/api/views.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ class AdDecisionView(GeoIpMixin, APIView):
         :<json string force_campaign: Limit results to ads from a specific campaign
 
         :>json string id: The advertisement slug of the chosen ad
-        :>json string text: The text of the ad
+        :>json string text: **Deprecated**:The HTML text of only the ad.
```
Comment (davidfischer) [inline adserver/api/views.py:71]: Do we actually want to deprecate this? I don't think we do. I suspect a lot of people (and probably our ad client) will still want to just do something like `$("#div").html(resp.text)`. I just think that having the text be simply a put together version of the body and the headline is ideal.

Comment (ericholscher) [reply adserver/api/views.py:71]: The client is using the `html` in the response -- which I assume is what most people would want if they are just plopping HTML in?

Comment (ericholscher) [reply adserver/api/views.py:71]: I'm happy not to deprecate it for now tho.

Comment (davidfischer) [reply adserver/api/views.py]: You're right! Maybe `text` should just be plain text or maybe we should just get rid of it.

Comment (ericholscher) [reply adserver/api/views.py]: Yea, I think people are using it currently, but we should definitely remove once we fully migrate.

**Pull request merged**